docs/dev/dev-mode.md

```diff
@@ -13,70 +13,103 @@ Description: Developer mode (dev mode) is intended to be used for testing and de
 
 Developer mode (dev mode) is intended to be used for testing and development purposes.
 
-This video shows the dev mode installation.
-
-<iframe width="950" height="475" src="https://www.youtube.com/embed/TG0GaAD_6J4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-
 ## Requirements
 
+- The Kubernetes node must pass the [host-check](https://raw.githubusercontent.com/harvester/harvester/master/hack/host-check.sh)
+- If the Kubelet's RootDir is not `/var/lib/kubelet`, it must be mounted bind to `/var/lib/kubelet`
+   ```bash
+   KUBELET_ROOT_DIR="replace with your kubelet root dir"
+   echo "${KUBELET_ROOT_DIR} /var/lib/kubelet none bind 0 0" >> /etc/fstab
+   mkdir -p /var/lib/kubelet && mount -a
+   ```
+- The Kubernetes controller-manager args requires `--use-service-account-credentials=true`
+- To display the CPU, memory, and disk usage of the host properly, the [metrics-server](https://github.com/kubernetes-sigs/metrics-server) must be installed in the cluster.
```

Comment:
```suggestion
- The Kubernetes node must pass the [host-check](https://raw.githubusercontent.com/harvester/harvester/master/hack/host-check.sh)
- If the Kubelet's RootDir is not `/var/lib/kubelet`, you must create a `bind mount` to `/var/lib/kubelet` as follow:
   ```bash
   KUBELET_ROOT_DIR="path to your kubelet root dir"
   echo "${KUBELET_ROOT_DIR} /var/lib/kubelet none bind 0 0" >> /etc/fstab
   mkdir -p /var/lib/kubelet && mount -a
   ```
- The Kubernetes controller-manager args requires `--use-service-account-credentials=true`
- To display the CPU, memory, and disk usage of the host properly, the [metrics-server](https://github.com/kubernetes-sigs/metrics-server) must be installed in the cluster.
```

Reply:
For the `Kubernetes controller-manager` point, can we add an example or a reference which explains how/where it should be set

Reply:
Small nit, line 19: "as follows:"

Reply:
corrected, thank you

Reply:
The controller-manager configuration is generally correct unless it has been specifically modified